pom.xml

```diff
@@ -268,6 +271,31 @@
                 </configuration>
             </plugin>
 
+            <plugin>
+                <groupId>org.jacoco</groupId>
+                <artifactId>jacoco-maven-plugin</artifactId>
```

Comment:
Could you explain the benefits of using jacoco maven plugin?

Reply:
- The most used and the community is active

- Has enough [functions](https://openclover.org/doc/manual/4.2.0/general--comparison-of-code-coverage-tools.html#Comparisonofcodecoveragetools-legend) to maintain `mop` project

- I tried other tool like `cobertura` but it parses [lambda expression error](https://github.com/streamnative/mop/runs/4296458283?check_suite_focus=true)
![image](https://user-images.githubusercontent.com/10069311/143142144-7213b2b7-5e77-4c8b-a599-00a0deb27f32.png) . The same issues are https://github.com/cobertura/cobertura/issues/176 https://github.com/stripe/stripe-java/pull/891